services/brig/src/Brig/API/Internal.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE TypeApplications #-}
```

Comment:
this should be in the cabal file as language defaults.  if it's not, please add it there.

Reply:
I've removed that. I've also noticed that brig has a lot of redundant language extensions, and some extensions that are used a lot like GeneralisedNewtypeDeriving. Maybe a conversation should be had about enabling them by default?